index.html

```diff
@@ -59,7 +59,7 @@
         <h3>Usability</h3>
 
         <details>
-          <summary>Sandstorm is the easiest way there has ever been to run a server. Install apps as easily as on your phone.</summary>
+          <summary>Install web apps like EtherCalc or WordPress with a click. Use our hosting, or install the platform easily on your own Linux server.</summary>
```

Comment:
(My mention of one of my reviewers is not intended to be a super serious rationale; I only really did it to poke fun at the he-said she-said nature of arguing over content for websites.)

I agree you did a great job compacting them; I just also think a lot of information was lost, so I think it's OK to restore a tiny bit of at the cost of one line.